docs/lsm/quickstart/quickstart.rst

```diff
@@ -82,13 +82,18 @@ This folder contains a project.yml, which looks like this:
     copyright: 2022 Inmanta
     modulepath: libs
     downloadpath: libs
-    repo:
-    - type: package
-        url: https://packages.inmanta.com/public/quickstart/python/simple/
-    - type: package
-        url: https://packages.inmanta.com/<token>/inmanta-service-orchestrator-|version_major|-stable/python/simple/
     install_mode: release
-    requires:
+    # The repo block is for ISO6, inmanta-core<=10.0
+    # This example requires licensed modules,
+    # replace <token> with inmanta access token you received with your license
+    repo:
+      - type: package
+        url: https://packages.inmanta.com/<token>/inmanta-service-orchestrator-6-stable/python/simple/
+    # The pip block is for ISO7 and up, inmanta-core>10.0
+    # This example requires licensed modules,
+    # replace <token> with inmanta access token you received with your license
+    pip:
+      index_url: https://packages.inmanta.com/<token>/inmanta-service-orchestrator-|version_major|-stable/python/simple/
```

Comment:
This PR goes into master and iso7, so there is no need to mention the config for iso6 here.

Reply:
I though we wanted to stay as close as possible to the content of this file in the example repo which contains this part. I removed it.